### ISSUE TITLE
Lube Fired From Fire Extinguishers does not lube up the Floor

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#17434]
(WEB REPORT BY: anglosphere REMOTE: 172.93.109.202:7777)
# Revision
f7a79805f577f465cecd6003e4766de37690023c

# Description
Lube fired from a fire extinguisher does not cover tiles in lube. 

# Steps to Reproduce
Get Fire Extinguisher
Put Lube in it
Close the cap
Fire it 

# What you Expected
Lube to cover where I fired at
# What Actually Happened
Lube not covering the floor.

**Comment by CptWad** [comment]
good

**Comment by ghost** [comment]
Yeah Anglo, if you can be arsed, the answers probably in the reagent distribution on covering floors effect system being probably spread too thin or failing to meet a min. Never looked into it past just use a 10u transfer or smoke.

**Comment by Kurfursten** [comment]
That seems likely. Here are the thresholds:
>Fuel: 0
>Carbon: 0 (makes cleanable decal)
>Vaporsalt: 0
>Bleach: 0
>Plantbgone: 0 (kills fungi attached to turf)
>Frostoil: 0 (damages slimes and slimepeople on turf)
>Four: 0
>Anthrazene: 0
>Space Cleaner: 1u
>Lube: 1u
>Water: 3u
>Radium: 3u
>Uranium: 3u
>Vomit: 3u
>Holywater: 5u
>Thermite: 5u (only applies to thermitable stuff ofc)

Cornoil is a special case. It requires 3u to make the floor slippery but 0u to cool off fires by 2000 degrees or half, whichever is less (who knew this..?)

**Comment by Pathid** [comment]
Cornoil's `reaction_turf` is copypasted from water's.

**Comment by Pathid** [comment]
Was just looking at the code for extinguishers and it appears that per attack they fire 5 overlapping sprays containing 1u each. Since lube requires a minimum of 1u _in one application_ to work, the effect of this is that if the lube is mixed with anything, even a tiny amount of other reagents, it will not work in extinguishers.
